cmd/litcli/ln.go

```diff
@@ -0,0 +1,751 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/lightninglabs/taproot-assets/asset"
+	"github.com/lightninglabs/taproot-assets/rfqmsg"
+	"github.com/lightninglabs/taproot-assets/taprpc"
+	"github.com/lightninglabs/taproot-assets/taprpc/rfqrpc"
+	tchrpc "github.com/lightninglabs/taproot-assets/taprpc/tapchannelrpc"
+	"github.com/lightningnetwork/lnd/cmd/commands"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/record"
+	"github.com/urfave/cli"
+)
+
+const (
+	// minAssetAmount is the minimum amount of an asset that can be put into
+	// a channel. We choose an arbitrary value that allows for at least a
+	// couple of HTLCs to be created without leading to fractions of assets
+	// (which doesn't exist).
+	minAssetAmount = 100
+)
+
+var lnCommands = []cli.Command{
+	{
+		Name:     "ln",
```

Comment:
would `tap` be better? `ln` to me is general `lightning network`

Reply:
Yeah, I agree. Though I think this is temporary and we'll remove them in a future release anyway. But for now I think Polar already uses the commands in some demos, so I don't want to rename them just for renaming sake.

Reply:
I suppose since it's inducing ln functionality but pairing with tapd it's a bit of both